animation_nodes/nodes/mesh/set_bevel_vertex_weight.py

```diff
@@ -0,0 +1,31 @@
+import bpy
+from ... data_structures import VirtualDoubleList
+from ... base_types import AnimationNode, VectorizedSocket
+
+class BevelVertexWeights(bpy.types.Node, AnimationNode):
+    bl_idname = "an_SetBevelVertexWeight"
+    bl_label = "Set Bevel Vertex Weight"
+    errorHandlingType = "EXCEPTION"
+
+    useWeightList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput("Object", "Object", "object", defaultDrawType = "PROPERTY_ONLY")
+        self.newInput(VectorizedSocket("Float", "useWeightList",
+            ("Weight", "weight"), ("Weights", "weights")))
+
+        self.newOutput("Object", "Object", "object")
+
+    def execute(self, object, weights):
+        if object is None: return None
+
+        if object.type != "MESH" or object.mode != "OBJECT":
+            self.raiseErrorMessage("Object is not in object mode or is no mesh object.")
```

Comment:
Separate those conditions into their own exceptions.

Reply:
Changed.

Reply:
@3DSinghVFX  `Object is no mesh object.` -> `Object is not a mesh object.`

Reply:
Omar, Mesh Object Output node also has `Object is not in object mode or is no mesh object` sentence. Should I also changed that too?

Reply:
No. We shouldn't add different things in the same pull request.

Reply:
Okay.